views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
+    <p>Why: {{event.image}}</p>
+  </div>
+  
+  <div>
+      <p>Do you want to partake in our revelry? Then RSVP!</p>
+      <form> 
+          <method="POST"></method>
```

Comment:
This is not the correct way to set the form method.  See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form.